### DISCUSSION TITLE
Found a typo in data.js

### DISCUSSION THREAD

**Discussion by danaildichev** [SupaGruen/SD-CheatSheet#3] [General]
Hello,

Thank you for making this. I've enjoyed playing around with it. The entry for "Forbes, Stanhope" lists "Irland" instead of Ireland in categories

```
{
        "Type": "1",
        "Name": "Forbes, Stanhope",
        "Born": "1857",
        "Death": "1947",
        "Prompt": "style of Stanhope Forbes",
        "NPrompt": "",
        "Category": "Painting, Oil, Landscape, Irland, 19th Century",
        "Extrainfo": "",
        "Checkpoint": "Deliberate 2.0",
        "Image": "Stanhope_Forbes.webp",
        "Creation": "202304061422"
    },
```

**Comment by danaildichev** [comment]
I think it's safe to change 
```
Children\\'s Book to Children's Book
```

but are you doing that on purpose because of whatever you're using Encode() for from the spaghetticode.js

**Reply by SupaGruen** [reply]
Yes, that's on purpose (encoding/decoding JSON), masking with one slash should normally be enough. Might also be related to the workflow from creating/editing the styles with my php based backend. I'll have to take a look at it when I have the time and a fresh head.
Fixed the two typos, thanks!

**Comment by danaildichev** [comment]
Under "Bonestell, Chesley" there's a typo in categories:

```
Sc-iFi should be be Sci-Fi
```
